### PR TITLE
Support for newer versions of OZ

### LINKED ISSUE #2271: Support for newer versions of accounts
### Which components does the task require to be changed? (think hard pls)

sncast

### Description

OZ/Argent/Braavos need to be up-to-date, we seem to have outdated class hashes on OZ/Argent right now. 
We need to either version those as well, but first - probably we would like to update them to the latest possible versions.
Argent seems to be up-to-date (to verify)
OZ was reported to be outdated
Braavos accounts was recently implemented, but it's worth to check the versions.

### PR DESCRIPTION
**Pull request by ksew1**
Closes #2271 

## Introduced changes

- Newest class hash for OpenZeppelin account contracts
- Test fixes

## Checklist

- [x] Linked relevant issue
- [ ] Updated relevant documentation
- [x] Added relevant tests
- [x] Performed self-review of the code
- [x] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/starknet/account.md [hunk 1]
```diff
@@ -102,15 +103,15 @@ For a detailed CLI description, see [account delete command reference](../append
 
 ### Custom Account Contract
 
-By default, `sncast` creates/deploys an account using [openzeppelin contract's class hash](https://starkscan.co/class/0x058d97f7d76e78f44905cc30cb65b91ea49a4b908a76703c54197bca90f81773).
+By default, `sncast` creates/deploys an account using [openzeppelin contract's class hash](https://starkscan.co/class/0x00e2eb8f5672af4e6a4e8a8f1b44989685e668489b0a25437733756c5a34a1d6).
```
Comment (tomek0123456789) [inline docs/src/starknet/account.md:106]:
```suggestion
By default, `sncast` creates/deploys an account using [OpenZeppelin's contract class hash](https://starkscan.co/class/0x00e2eb8f5672af4e6a4e8a8f1b44989685e668489b0a25437733756c5a34a1d6).
```

**Review by tomek0123456789 — APPROVED** [review]
:shipit:

File: docs/src/starknet/account.md [hunk 2]
```diff
@@ -102,15 +103,15 @@ For a detailed CLI description, see [account delete command reference](../append
 
 ### Custom Account Contract
 
-By default, `sncast` creates/deploys an account using [openzeppelin contract's class hash](https://starkscan.co/class/0x058d97f7d76e78f44905cc30cb65b91ea49a4b908a76703c54197bca90f81773).
+By default, `sncast` creates/deploys an account using [OpenZeppelin's contract class hash](https://starkscan.co/class/0x00e2eb8f5672af4e6a4e8a8f1b44989685e668489b0a25437733756c5a34a1d6).
```
Comment (Arcticae) [inline docs/src/starknet/account.md:106]:
```suggestion
By default, `sncast` creates/deploys an account using [OpenZeppelin's account contract class hash](https://starkscan.co/class/0x00e2eb8f5672af4e6a4e8a8f1b44989685e668489b0a25437733756c5a34a1d6).
```

File: docs/src/starknet/account.md [hunk 3]
```diff
@@ -102,15 +103,15 @@ For a detailed CLI description, see [account delete command reference](../append
...
 It is possible to create an account using custom openzeppelin contract declared to starknet. This can be achieved
```
Comment (Arcticae) [inline docs/src/starknet/account.md:107]: btw: Doesn't this work with other accounts?

**Review by Arcticae — APPROVED** [review]

File: docs/src/starknet/account.md (hunk 3 above)
Comment (ksew1) [reply docs/src/starknet/account.md:107]: Yes, updated that

**Pull request merged**